practice_app/pages/api/getTranslationModule.js

```diff
@@ -0,0 +1,803 @@
+exports.checkTranslationResponse = function (response, sourceLanguage, targetLanguage, originalText) {
+    const translation = response.data[0][0][0]
+    const sourceText = response.data[0][0][1]
+    const sourceLang = getSourceLangName(response.data[2])
+    const isAlternativeSourceExist = response.data[8] ? response.data[8][0].length > 1 : false
+
+    const alternativeSourceLangs = []
+    if (isAlternativeSourceExist) {
+        let i
+        for (i = 0; i < response.data[8][0].length; i++) {
```

Comment:
Declare `let i` inside the initialization of the for loop, so `for(let i = 0; ... )`

Reply:
Fixed.